core/src/main/java/forklift/message/Header.java

```diff
@@ -161,14 +161,15 @@
     Type("JMSType", "type", "message-annotations.x-opt-jms-type", String.class);
 
 
-    private String jmsMessage;
-    private String stompMessage;
-    private String amqpMessage;
-    private Type headerType;
+    final private String jmsMessage;
+    final private String stompMessage;
+    final private String amqpMessage;
```

Comment:
normally the final qualifier would come after private based on some arbitrary style I'm trying to uphold. :\